experiences/multicard/multicard.test.js

```diff
@@ -87,18 +87,18 @@ const multicardTest = function() {
 		expect(isAtActiveSnapPoint).equal(true);
 	});
 
-	it('remove card, should not be displayed in viewport', async function() {
-		await removeCard(
-			browser,
-			config.cards.nextArticle.cardSelector
-		);
+	// it('remove card, should not be displayed in viewport', async function() {
+	// 	await removeCard(
+	// 		browser,
+	// 		config.cards.nextArticle.cardSelector
+	// 	);
 
-		const firstCard = await browser.$(config.cards.nextArticle.cardSelector);
+	// 	const firstCard = await browser.$(config.cards.nextArticle.cardSelector);
 
-		const firstCardIsInViewport = await firstCard.isDisplayedInViewport();
+	// 	const firstCardIsInViewport = await firstCard.isDisplayedInViewport();
 
-		expect(firstCardIsInViewport).equal(false);
-	});
+	// 	expect(firstCardIsInViewport).equal(false);
+	// });
```

Comment:
Is this intended or it was committed by mistake?

Reply:
if intended better to go `it.skip` as that will be part of jest output, rather than commented code for which we lose track